packages/snowpack/assets/hmr.js

```diff
@@ -146,6 +148,7 @@ socket.addEventListener('message', ({data: _data}) => {
   }
   debug('message: update', data);
   debug(data.url, Object.keys(REGISTERED_MODULES));
```

Comment:
Related: This line can be removed completely, and is only useful for us to troubleshoot / not useful to the end user